File: templates/java-webflux/subsystem/post/component/src/test/java/cobalt/post/MockData.java
package cobalt.post;

import cobalt.post.model.Post;
import java.util.List;

public class MockData {
  static final List<Post> posts;

  static {
    posts = List.of(
      new Post(1, 1, "Lorem ipsum dolor sit amet.", "Curabitur ultrices lacus lorem, vitae tincidunt lorem convallis."),
      new Post(2, 1, "Maecenas maximus sodales nibh vel.", "Pellentesque scelerisque tortor nec lorem volutpat, sed semper."),
      new Post(3, 3, "Sed tempus, justo at viverra.", "Cras efficitur, nunc eu maximus congue, magna quam.")
    );
  }

}

File: templates/kotlin-webmvc/subsystem/post/component/src/test/kotlin/{{PACKAGE_NAME}}/post/MockData.kt
package {{PACKAGE_NAME}}.post

import {{PACKAGE_NAME}}.post.model.Post

object MockData {
  val posts = listOf(
    Post(id = 1, userId = 1, title = "Lorem ipsum dolor sit amet.", body = "Curabitur ultrices lacus lorem, vitae tincidunt lorem convallis."),
    Post(id = 2, userId = 1, title = "Maecenas maximus sodales nibh vel.", body = "Pellentesque scelerisque tortor nec lorem volutpat, sed semper."),
    Post(id = 3, userId = 3, title = "Sed tempus, justo at viverra.", body = "Cras efficitur, nunc eu maximus congue, magna quam.")
  )
}